src/components/InspectorExtensionManager.js

```diff
@@ -0,0 +1,38 @@
+export default class InspectorExtensionManager {
```

Comment:
If this is only housing static methods, is there any advantage to having them in a class as oppose to exporting them as standalone functions? Similar to how we have our `/tests/e2e/support/utils.js` file set up? (Though that file is growing a bit big now).

Keeping them in a class like this means we always have to import the entire class, and we have the extra overhead/syntax of a class to worry about.

Reply:
I agree and I think the fact that all of the methods are static is the smell that gives this away. Let me change the code to this, it's a better structure.